docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -16,7 +16,7 @@ Current binder implementations include:
 The Spring Cloud Stream Binder for Azure Event Hub provides the binding implementation for the Spring Cloud Stream.
 This implementation uses Spring Integration Event Hub Channel Adapters at its foundation. From design's perspective,
 Event Hub is similar as Kafka. Also, Event Hub could be accessed via Kafka API. If your project has tight dependency
-on Kafka API, you can try link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0/eventhubs/spring-cloud-azure-starter/spring-cloud-azure-sample-eventhubs-kafka[Event Hub with Kafka API Sample]
+on Kafka API, you can try link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0.0-beta.3/eventhubs/spring-cloud-azure-starter/spring-cloud-azure-sample-eventhubs-kafka[Event Hubs with Kafka API Sample]
```

Comment:
I remember Rujun did some change to use variable for the version, please investigate whether this can be changed to use variable https://github.com/microsoft/spring-cloud-azure/pull/817/files.